app/controllers/SubmissionsController.scala

```diff
@@ -45,4 +49,15 @@ class SubmissionsController @Inject()(
     } yield Ok(submissionsPage(result.toSeq))
   }
 
+  def amend(id: String): Action[AnyContent] = authenticate.async { implicit request =>
+    customsDeclareExportsConnector.findDeclaration(authenticatedSessionId, id) flatMap {
+      case Some(declaration) =>
+        exportsCacheService
+          .update(authenticatedSessionId, declaration)
```

Comment:
is here possible `sessionId` and `declaration.sessionId` could be different ?

Reply:
No, session id isn't stored in the back end so the only way it could be populated is from the request

Reply:
I'm not sure we actually need `declaration.sessionId` any more to be honest

Reply:
sessionId was and old `Id` for declaration draft